tests/serviceBookTests.groovy

```diff
@@ -30,12 +33,10 @@ class ServiceBookTests extends BasePipelineTest {
         }
     }
 
-    // @Test
-    // void projectWithTests() {
-    //     def resp = [body:'{"data": [{"foo": {"tests": [{"name": "bar", "jenkins_pipeline": true}]}}]}']
-    //     def script = loadScript('src/org/mozilla/fxtest/ServiceBook.groovy')
-    //     script.doGetHttpRequest = { String url -> [resp] }
-    //     def tests = script.getProjectTests('foo')
-    //     assert tests != null
-    // }
+    @Test
+    void projectWithTests() {
+        def script = loadScript('src/org/mozilla/fxtest/ServiceBook.groovy')
+        def tests = script.getProjectTests('kinto')
+        assert tests != null
```

Comment:
This is great. I wonder if we should verify that this is a non-empty list, and include negative tests for a non-existant project and a project with no pipeline tests.